airbyte-server/src/test/java/io/airbyte/server/handlers/WebBackendCheckUpdatesHandlerTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.airbyte.api.model.generated.WebBackendCheckUpdatesRead;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.server.services.AirbyteGithubStore;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WebBackendCheckUpdatesHandlerTest {
+
+  ConfigRepository configRepository;
+  AirbyteGithubStore githubStore;
+  WebBackendCheckUpdatesHandler webBackendCheckUpdatesHandler;
+
+  final static boolean INCLUDE_TOMBSTONE = false;
+
+  @BeforeEach
+  void beforeEach() {
```

Comment:
Nit: this could be replace by the annotation @Mock and @InjectMock

Reply:
Played with the annotations, I think a setup function feels clearer than having annotations on both members + class to figure out how the objects are initialized.
Might be related to the fact that I am not a big fan of the lombok type of annotations.